.github/workflows/pypi-build-artifacts.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ ubuntu-22.04, windows-2022, macos-13, macos-14, macos-15 ]
+        os: [ ubuntu-22.04, windows-2022, macos-13, macos-14 ]
```

Comment:
What's up with `macos-15`? :)

Reply:
macos-14, macos-15 are the same hardware so produce the same artifacts. i was getting some error uploading to pypi due to 
```
pypi zipfile.BadZipFile: Bad magic number for central directory
```
removing macos-15 worked 🤷

Reply:
Hmm, let's see if we can restore `macos-15` at a later moment